### DISCUSSION TITLE
Not importing UPFD dataset

### DISCUSSION THREAD

**Discussion by fffahim** [pyg-team/pytorch_geometric#2710] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45016119/121134441-36acc600-c855-11eb-85d2-ade81ba14388.png)

why this error is occurring all the time? i am using google colab. i will be grateful if anyone can help
Thank you

**Comment by rusty1s** [comment, marked as answer]
It looks like you have still a previous PyG version installed (`1.3.1`), which does not contain `UPFD` yet. Try running your installation command via:
```
pip install --upgrade git+https://github.com/rusty1s/pytorch_geometric.git
```

**Reply by fffahim** [reply]
Yes it works. Thank you so much. God bless you
